README.md

```diff
@@ -1,19 +1,40 @@
+<span style="color:red"> **WORKING DRAFT - These documents are a working draft and are not official GSA Policy** </span>
+
 # GSA API Standards
 
-This document captures **GSA's recommended best practices and standards**. We encourage GSA development groups to used these standards when developing APIs.
+This document captures **GSA's recommended best practices and standards for Application Programming Interfaces (APIs)**. We encourage GSA development groups to use these standards when developing APIs for GSA.
+
+# Index
+[About These Standards](#about-these-standards)  
+[Overall Considerations](#overall-considerations)  
+[Developers Are Your End Users](#developers-are-your-end-users)  
+[Design Considerations](#design-considerations)  
+[Future Topics](#future-topics)  
+[Public Domain](#public-domain)
+
+
+## About These Standards
+
+These standards are forked from the [18F API Standards](https://github.com/18F/api-standards). They are also influenced by several other sources, including the [White House API Standards](https://github.com/WhiteHouse/api-standards), and several from the private sector.
```

Comment:
Response: no I removed to avoid giving the impression of endorsing private companies.